descwl_shear_sims/render_sim.py

```diff
@@ -46,9 +46,9 @@ def append_wcs_info_and_render_objs_with_psf_shear(
     shear_scene : bool
         If True, the object positions and their shapes are sheared. Otherwise,
         only the object shapes are sheared.
-    expand_star_stamps: bool
-        If True, expand bright star stamps to avoid rendering issues, default
-        True
+    threshold: float
+        For bright stars we will calculate the radius at which we reach
```

Comment:
Doc string is wrong. I think it does it for all objects.

Reply:
there is a mag cut at 18

Reply:
but that is incorrect grammar :)